app/views/books/_review.html.erb

```diff
@@ -0,0 +1,7 @@
+<div>
+  <p class="review"><%= @review.body %></p>
+  <% if @review.user %>
```

Comment:
If I'm thinking about a Review in real life, I wouldn't want a review to be created without a user, you need a user for them to write a review.

Reply:
Overall I probably wouldn't default to using a partial. Partials are good for DRYing up our code, but only if we have a need for the same chunk of view twice.